addons/website/static/src/xml/website.editor.xml

```diff
@@ -108,19 +108,6 @@
             </div>
         </t>
     </t>
-    <!-- Anchor Name option dialog -->
-    <div t-name="website.dialog.anchorName">
-        <div class="form-group row">
-            <label class="col-form-label col-md-3" for="anchorName">Choose an anchor name</label>
-            <div class="col-md-9">
-                <input type="text" class="form-control o_input_anchor_name" id="anchorName" t-attf-value="#{currentAnchor}" placeholder="Anchor name"/>
-                <div class="invalid-feedback">
-                    <p class="d-none o_anchor_not_valid">The chosen name is not valid (use only a-Z A-Z 0-9 - _)</p>
-                    <p class="d-none o_anchor_already_exists">The chosen name already exists</p>
-                </div>
-            </div>
-        </div>
-    </div>
```

Comment:
Mmh... I don't know how the dialog could work without this... are you sure you tested your work in this final state? :wink: Maybe you still had the template in cache :wink: 

(note: this should stay but without the not_valid message etc)